docs/README-DOCS.md

```diff
@@ -0,0 +1,44 @@
+# Documenting nbconvert
```

Comment:
I'd be inclined to call this file just `README.md` instead of `README-DOCS.md`. Github will display `README.md` underneath the files list when you view the docs folder, but it doesn't do anything with `README-DOCS.md`.